Nakama/TinyJson/JsonParser.cs

```diff
@@ -169,9 +170,17 @@ private static object ParseValue(Type type, string json)
         {
             if (type == typeof(string))
             {
+                // Return the raw value if it is unquoted (e.g. a number)
+                if (json[0] != '"')
+                {
+                    return json;
```

Comment:
Okay could we move this `if` block underneath the length check, and could we check that the beginning _and_ end contain quotes?

Reply:
We can add a check that the end contains quotes for sure. But we can't move it to below the length check incase the number is i.e. `1` or `12`.

Reply:
I see -- what about just guarding against the out-of-bounds checks in the quotation checks?

Reply:
I see. Could you guard against out-of-bounds checks then in the if-statement expressions that are checking for quotes?